### PR TITLE
Info strip - add fullWidth prop

### PR DESCRIPTION
**Pull request by omerhochbob**
Closes https://app.asana.com/0/1199672671179462/1200451982344589/f

We need it for the new Employee Acknowledgement feature. ([Figma link](https://www.figma.com/file/6FecfMlTJMAZjHrfdaTGSm/My-Reviews?node-id=545%3A9265))

Screen Shot 2021-06-13 at 10 36 46
Screen Shot 2021-06-13 at 10 36 50

After radius fix:
Screen Shot 2021-06-13 at 11 50 22

### CODE REVIEW

**Review by atarab1 — APPROVED** [review]

File: projects/ui-framework/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bob-style",
-  "version": "4.3.275",
+  "version": "4.3.276",
```
Comment (atarab1) [inline projects/ui-framework/package.json:3]: i think we should not update the package

Comment (omerhochbob) [reply projects/ui-framework/package.json:3]: @atarab1 it is an automatic commit added by bot in every PR in bob-style

File: projects/ui-framework/src/lib/indicators/info-strip/info-strip.component.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ import { InfoStripIconSize, InfoStripIconType } from './info-strip.enum';
 export class InfoStripComponent {
   @HostBinding('attr.data-type') @Input() iconType: InfoStripIconType =
     InfoStripIconType.information;
+  @HostBinding('attr.full-width') @Input() fullWidth = false;
```
Comment (galloween) [inline projects/ui-framework/src/lib/indicators/info-strip/info-strip.component.ts:22]: custom attributes should be prefixed with 'data-'

File: projects/ui-framework/src/lib/indicators/info-strip/info-strip.component.html [hunk 3]
```diff
@@ -3,7 +3,10 @@
         [size]="iconSize">
 </b-icon>
 
-<div class="content flx-col flx-grow flx-col-align-y">
+<div class="flx content flx-grow"
+     [ngClass]="fullWidth
+        ? 'flx-row flx-row-align-y full_width'
```
Comment (galloween) [inline projects/ui-framework/src/lib/indicators/info-strip/info-strip.component.html:8]: css classes should use kebab-case

File: projects/ui-framework/src/lib/indicators/info-strip/info-strip.component.html [hunk 4]
```diff
@@ -12,5 +15,6 @@
 
   <b-link *ngIf="link"
           [config]="link"
-          (clicked)="linkClicked.emit()"></b-link>
+          (clicked)="linkClicked.emit()"
+          [ngClass]="fullWidth && 'mrg-l-8'"></b-link>
```
Comment (galloween) [inline projects/ui-framework/src/lib/indicators/info-strip/info-strip.component.html:19]: for single conditional classes, this notation is a bit more readable:  
 `[class.mrg-l-8]="fullWidth"`

**Review by galloween — COMMENTED** [review]
see comments, mainly about names/conventions

**Review by galloween — APPROVED** [review]

**Pull request merged**